cmd/osmosisd/cmd/balances_from_state_export.go

```diff
@@ -177,18 +177,22 @@ Example:
 			}
 
 			authGenesis := authtypes.GenesisState{}
-			clientCtx.JSONCodec.MustUnmarshalJSON(genState["auth"], &authGenesis)
-			accounts, err := authtypes.UnpackAccounts(authGenesis.Accounts)
-			if err != nil {
-				panic(err)
-			}
-			accounts = authtypes.SanitizeGenesisAccounts(accounts)
+			clientCtx.Codec.MustUnmarshalJSON(genState["auth"], &authGenesis)
+
+			// This didn't seem to do anything, so have removed.
+			/*
+				accounts, err := authtypes.UnpackAccounts(authGenesis.Accounts)
+				if err != nil {
+					panic(err)
+				}
+				sanitizedAccounts := authtypes.SanitizeGenesisAccounts(accounts)
+			*/
```

Comment:
Hrmm, can we undo a lot of changes to this file. I think they're going to conflict with other ongoing work.

I may make a commit for this later

Reply:
I can do like

git checkout main -- cmd/osmosisd/cmd/balances_from_state_export.go